codec-http/src/main/java/io/netty/handler/codec/http/websocketx/extensions/WebSocketServerExtension.java

```diff
@@ -26,6 +26,7 @@
      *
      * @return the acknowledged extension configuration.
      */
+    //TODO: after migrating to JDK 8 rename this to 'newResponseData()' and mark old as deprecated with default method
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)